src/cpp/src/whisper_pipeline.cpp

```diff
@@ -13,6 +13,10 @@
 #include "whisper/whisper_feature_extractor.hpp"
 #include "whisper/whisper_models.hpp"
 
+#include "whisper_pipeline_base.hpp"
+#include "whisper_pipeline_static.hpp"
+
+// FIXME: Remove these duplicates
```

Comment:
Looks can't be simply moved to utils as there are already functions with such signature for llm pipeline